.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: "✨ Feature request"
+about: If you have an idea to improve an existing WordCamp feature, please let us know.
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
`MUST` - Feature requests need to be approved by the Community Team before we can accept them, so I think we should reflect that here.

`MAYBE` - We could just remove all of these sections and just ask people to follow this instead:

https://make.wordpress.org/community/feature-requests-for-community-sites/

...or, maybe we should add a warning (similar to security in the bug report template) to follow that process first, and then once it's been approved, then they can open an issue here, and the template should ask them to link to the p2 post where it was approved. 

If we do that, there shouldn't be a need to describe to problem or alternatives, since those should already be worked through in the p2 post. We'd only need a description of the solution that was agreed upon on the p2.

Reply:
> the template should ask them to link to the p2 post where it was approved

+1 for this.